sourcegraph/sourcegraph
# insights: support for pie charts in the GraphQL API

Closes https://github.com/sourcegraph/sourcegraph/issues/26675

## Description

The previous PR had to be reverted due to the API changes causing typescript errors: https://github.com/sourcegraph/sourcegraph/pull/27237

This new PR contains that previous one, plus the create and update mutations for pie charts. The functionality to create and update pie charts is somewhat simpler than line charts due to there being one fixed series, but otherwise there's a lot in common there.

Also a couple of temporary fixes for typescript errors introduced by this change.

## Testing Steps

Try creating and updating pie charts and see that those endpoints work as expected.
